spec/index.md

```diff
@@ -45,7 +54,12 @@ The micro:bit V2 powers both MCUs with all power sources, so to set the board in
 
 We want to define 4 user-facing power modes for the micro:bit board. These board-level modes will be achieved via a combination of different subsystem power modes.
 
-- **On Mode**: Normal running mode.
+- [On Mode](#on-mode)
+- [Deep Sleep Mode](#deep-sleep-mode)
+- [Off Mode](#off-mode)
+- [Stand-by Mode](#stand-by-mode)
```

Comment:
I think Jonny meant to add this to the introduction.

Reply:
I've reordered this so the introduction comes before the terminology and contains the set of links to the power modes. I think it reads better like this. What do you think?